analyzers/packaging/SonarAnalyzer.CSharp.Styling.nuspec

```
@@ -0,0 +1,33 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>SonarAnalyzer.CSharp.Styling</id>
+    <version>9.24.0.0</version>
+    <title>Internal styling analyzer for C#</title>
+    <authors>SonarSource</authors>
+    <owners>SonarSource</owners>
+    <license type="expression">LGPL-3.0-only</license>
+    <projectUrl>https://redirect.sonarsource.com/doc/sonar-visualstudio.html</projectUrl>
+    <repository type="git" url="https://github.com/SonarSource/sonar-dotnet" />
+    <icon>images\sonarsource_64.png</icon>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>Internal styling analyzer for C#</summary>
+    <description>We do not provide support for this package.</description>
```

Let's set the expectations right.
```suggestion
    <description>This package contains a set of coding style rules to follow in Sonar code base. Coding style changes as well as breaking changes will appear between minor versions without prior notice. We do not provide support for this package.</description>
```